app/javascript/src/utils/index.js

```diff
@@ -1,3 +1,21 @@
 import * as R from "ramda";
 
+const camelize = str =>
+  str
+    .replace(/[^a-z0-9]/gi, " ")
+    .toLowerCase()
+    .split(" ")
+    .map((elem, index) =>
+      index === 0
+        ? elem
+        : elem[0].toUpperCase() + elem.substring(1, elem.length)
+    )
+    .join("");
+
 export const isPresent = R.pipe(R.either(R.isNil, R.isEmpty), R.not);
+
+export const buildSelectOption = optionsAsStringArray =>
```

Comment:
https://bigbinary.slack.com/archives/C01T5Q4DZAR/p1663250339031239?thread_ts=1663060378.971029&cid=C01T5Q4DZAR

```suggestion
export const buildSelectOptions = values =>
```

Reply:
I have made changes accordingly .Please check latest commit https://github.com/eshaanks/neeto-ui-challenge/pull/22/commits/97afceefee0c35b9f10eb95fd2e4696d87b95f7b.